gulp-tasks/lint.js

```diff
@@ -39,15 +39,16 @@ const lintPackage = (projectPath) => {
 
 gulp.task('lint:repo', () => {
   return gulp.src([
+    `!./packages/**/*`,
     // Gulp -tasks
-    `./*.js`,
+    `./**/*.js`,
     // Root level JS Files
     `../*.js`,
   ])
   .pipe(eslint())
   .pipe(eslint.format())
   .pipe(eslint.results((results) => {
-    if (results.errorCount > 0 || results.warningCount > 0) {
+    if (results.errorCount > 0) {
```

Comment:
Yup. The top level lib/directory wasn't being caught by the lint task. Once I had fixed that, there are JSDocs missing which is across a number of projects.

Suggest we allow building with warnings until we are at a further along stage and then clean house and enforce the no warnings if we want.